src/Command/Docker/Build.php

```diff
@@ -122,6 +123,15 @@ protected function configure()
                 null,
                 InputOption::VALUE_OPTIONAL,
                 'RabbitMQ version'
+            )->addOption(
+                self::OPTION_MODE,
+                'm',
+                InputOption::VALUE_OPTIONAL,
+                sprintf(
+                    'Mode of environment (%s)',
+                    implode(', ', [ComposeManagerFactory::COMPOSE_DEVELOPER, ComposeManagerFactory::COMPOSE_PRODUCTION])
+                ),
+                ComposeManagerFactory::COMPOSE_DEFAULT
```

Comment:
Why do you need COMPOSE_DEFAULT if it the same as COMPOSE_PRODUCTION?

Reply:
It is configurable constant to set default strategy

Reply:
Don't see a point to use additional constant which is using only in one particular place to set a default value.
Just put COMPOSE_PRODUCTION it will be more obvious. 
Because firstly I thought that there are 3 different modes 'default', 'developer' and 'production'

Reply:
Makes sense, fixed